frontend/public/components/cluster-overview.jsx

```diff
@@ -198,9 +198,18 @@ const GraphsPage = ({fake, limited, namespace, openshiftFlag}) => {
     return body;
   }
 
-  return <Firehose resources={[{kind: 'Namespace', name: namespace, isList: false, prop: 'data'}]}>
+  const resources = [{
```

Comment:
This is about all I changed, VSCode autoformat caused some whitespace updates.

Reply:
`?w=1`

Reply:
Can you disable autoformat?

Reply:
Sure can. Want me to rollback these whitespace changes?

I tend to think we could/should use an autoformatter.  We have the `.editorconfig` file already in the repo.

Reply:
Please rollback the whitespace changes, yeah.

We should only use an autoformatter if everyone is IMO. And it will cause merge conflicts when back-porting fixes.

I'd rather use eslint to enforce styles we care about when possible.